client-spark/spark-3/src/main/scala/org/apache/spark/shuffle/celeborn/ShuffleFetchFailureReportTaskCleanListener.scala

```diff
@@ -17,12 +17,26 @@
 
 package org.apache.spark.shuffle.celeborn
 
+import scala.collection.JavaConverters._
+
 import org.apache.spark.scheduler.{SparkListener, SparkListenerStageCompleted}
 
 class ShuffleFetchFailureReportTaskCleanListener extends SparkListener {
   override def onStageCompleted(stageCompleted: SparkListenerStageCompleted): Unit = {
-    SparkUtils.removeStageReportedShuffleFetchFailureTaskIds(
-      stageCompleted.stageInfo.stageId,
-      stageCompleted.stageInfo.attemptNumber())
+    val stageId = stageCompleted.stageInfo.stageId
+    val stageAttemptId = stageCompleted.stageInfo.attemptNumber()
+    val shuffleFetchFailureTaskIds =
+      SparkUtils.removeStageReportedShuffleFetchFailureTaskIds(stageId, stageAttemptId)
```

Comment:
even we do not know whether the `shuffleFetchFailureTaskIds` related tasks did cause the FetchFailedException eventually.